### PR TITLE
Use VarDumper in web profiler view

### PR DESCRIPTION
**Pull request by alcaeus**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
         "doctrine/dbal": "~2.3",
         "jdorn/sql-formatter": "~1.1",
         "symfony/doctrine-bridge": "~2.2|~3.0",
-        "doctrine/doctrine-cache-bundle": "~1.0"
+        "doctrine/doctrine-cache-bundle": "~1.0",
+        "symfony/web-profiler-bundle": "~2.7|~3.0"
```
Comment (Ocramius) [inline composer.json]: Doesn't this also require the web profiler bundle to be enabled?

Comment (alcaeus) [inline composer.json]: Yes, which is why I'm not sure whether we need to add this. On the other hand, we should make sure the `profiler_dump` twig function exists. Dropping Symfony 2.3 would probably implicitly ensure it is, but I'd rather have it explicit.

Comment (Ocramius) [inline composer.json]: > Dropping Symfony 2.3 would probably implicitly ensure it is

What's blocking this then? :-)

Comment (alcaeus) [inline composer.json]: Point taken - might as well remove it in this PR ;)

**Comment by kimhemsoe** [comment]
Requiring 2.7 is a no go as we do not want to maintain more then one version of doctrine bundle. Is there any way to archive this while keeping BC?

**Comment by Ocramius** [comment]
@kimhemsoe we can just drop 2.3 and that's it. There's no need to keep BC with it, unless I'm missing something obvious.

**Comment by kimhemsoe** [comment]
@Ocramius It is a policy set by @stof and i am not going change that without a OK from him atleast. 

From the looks of the it should be pretty simple to support 2.3 and still use VarDumper when it is available.

**Comment by alcaeus** [comment]
We have two choices here:
1. Drop support for 2.3 to at least somewhat ensure the `profiler_dump` function exists
2. Leave it as is and let people fail hard when updating the bundle in their Symfony 2.3 apps.

I'd argue for dropping 2.3 support here. Bugfixing has ended on May 2016, it will only receive security fixes for another 6 months. Should something arise, feel free to ping me and I'll personally deal with the overhead of releasing a security fix version for 1.6 in addition to the regularly scheduled release. Personally, I don't think we need to support 2.3 any longer.

**Comment by Nyholm** [comment]
I agree with dropping 2.3 if it causes pain/issues.

**Comment by Ocramius** [comment]
> Bugfixing has ended on May 2016

That's good enough for me: kill it, I'd say.

We fix security issues wherever possible/sensible anyway, although we don't have an explicit policy about it. DoctrineBundle is a relatively security-safe area, since it's just an integration layer, so I don't expect security fixes to land in here anyway.

**Comment by kimhemsoe** [comment]
Ok, if i do not get any objections in the next few days i will bump the required symfony version.

**Comment by Ocramius** [comment]
@kimhemsoe this PR does that, so merging would be sufficient, I suppose.

**Comment by alcaeus** [comment]
@kimhemsoe that sounds like a plan. I'd also look through the issues and PRs to identify any other things we can add before tagging 1.7.0.

**Comment by kimhemsoe** [comment]
@Ocramius Sorry.. i meant bumping by merging :D

**Comment by stof** [comment]
Now that 2.3 is EOLed, dropping support for it is indeed acceptable

**Comment by havvg** [comment]
What about changing the required version to `2.8` as it's the LTS already? I mean, `3.0` is also allowed. As per release notes `2.8` is the BC variant of `3.0`.

**Comment by alcaeus** [comment]
Symfony 2.7 is a LTS release as well and it's supported until May 2018. As much as I'd like to, I don't think dropping that is an option.

**Comment by havvg** [comment]
Oh, I just took a look on the roadmap, it states 2.8, but the details read 2.7 being LTS, too. Well, that makes my comment obsolete :-)

File: composer.json [hunk 2]
```diff
@@ -25,18 +25,18 @@
     ],
     "require": {
         "php": ">=5.3.2",
```
Comment (fabpot) [inline composer.json]: Can be bumped to 5.3.9 as well.

**Comment by fabpot** [comment]
👍

**Comment by stof** [comment]
thanks

**Pull request merged**